src/khoj/database/models/__init__.py

```diff
@@ -452,6 +462,17 @@ class ServerChatSettings(DbBaseModel):
         WebScraper, on_delete=models.CASCADE, default=None, null=True, blank=True, related_name="web_scraper"
     )
 
+    def clean(self):
+        error = {}
```

Comment:
Rather than throwing an error, let's correct it automatically? If the admin ends up here, it's most likely a mistake on their part, so we can gracefully correct it imo.